lighthouse-core/audits/predictive-perf.js

```diff
@@ -51,7 +53,11 @@ class PredictivePerf extends Audit {
   static getPessimisticFMPGraph(graph, traceOfTab) {
     const fmp = traceOfTab.timestamps.firstMeaningfulPaint;
     return graph.cloneWithRelationships(node => {
-      return node.endTime <= fmp;
+      if (node.endTime > fmp) return false;
+      // Include CPU tasks that performed a layout
+      if (node.type === Node.TYPES.CPU) return node.didPerformLayout();
```

Comment:
btw what's special about these?

Reply:
they're included in pessimistic estimate because they frequently capture large CPU work that was needed to render something meaningful and the resources that might not have been render-blocking that were required to perform it. example: a client-side rendered page with a large script in the body

Reply:
gotcha.. so i guess this is more semantically like `node.containsMeaningfulWork()` 

.. and in which case i'm wondering if there are any other work payloads besides layout that also fit into the definition. Style/Paint probably do, but I can imagine including them wouldn't change the results so much...